### ISSUE TITLE
Should NYCMesh have a splash page - nodogsplash or alternative

### ISSUE THREAD

**Issue by josmo** [nycmeshnet/nycmeshfeed#23]
Issue https://github.com/libremesh/lime-packages/issues/76 talks about not having splash pages and using other ways to be known

Currently the Libremesh ssid are -- NYCMesh.net xxx -- to be at the beginning and make it more clear where folks can get information.

I tend to agree on not needing splash pages and it being a bit of a pain all the way around, but should discuss.

**Comment by bhny** [comment]
yes it is a pain but without it people barely know what wifi they are using other than it is "free"

It should be easy to turn off for apartment installs, but it is really good advertising for public access points.
